core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -117,6 +122,11 @@ public TriviaButtonBuilder() {
     incorrectButtons = new Array<TextButton>();
     for (int i = 0; i < currentQuestion.incorrectAnswers.length; i++) {
       incorrectAnswerButton = new TextButton(currentQuestion.incorrectAnswers[i], textButtonStyle);
+      incorrectAnswerButton.getLabelCell().width(300);
+      incorrectAnswerButton.setWidth(300);
+      incorrectAnswerButton.getLabelCell().center();
+      incorrectAnswerButton.getLabel().setWrap(true);
+      incorrectAnswerButton.invalidate();
```

Comment:
There seems to be some code duplication between correctAnswerButton and the incorrectAnswerButtons.  Consider pulling the previous 5-6 lines into a private helper method.